l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -1405,6 +1417,20 @@ def _set_invoice_lines(self, product, invoice_line_data, invoice_lines,
         if product:
             invoice_line_data['product_id'] = product.id
             self.adjust_accounting_data(product, invoice_line_data)
+        account_id = invoice_line_data['account_id']
```

Comment:
qui l'account_id c'è sempre visto che non fai .get() ?

Reply:
Grazie della review!
Non avevo ancora finito (per questo era in _Draft_), ora questa parte è cambiata e il problema non dovrebbe porsi :)

Ti faccio sapere appena è pronta per review